doc-content/enterprise-only/bpmn/bpmn-subprocesses-ref.adoc

```diff
@@ -1,87 +1,54 @@
+An embedded subprocess encapsulates a part of the process. It must contain a start event and at least one end event. Note that the element enables you to define local subprocess variables that are accessible to all elements inside this container.
 
 [[_adhoc_sub_process]]
-.AdHoc Sub-Process
-COMMENT: edit 
-"`An Ad-Hoc Sub-Process is a specialized type of Sub-Process that is a group of Activities that have no REQUIRED sequence relationships. A set of Activities can be defined for the Process, but the sequence and number of performances for the Activities is determined by the performers of the Activities. footnote:[Business Process Model and Notation (BPMN). Version 2.0, OMG Document Number: formal/2011-01-03 http://www.omg.org/spec/BPMN/2.0]`"
+.AdHoc subprocess
 
+An ad-hoc subprocess or process contains a number of embedded inner activities and is intended to be executed with a more flexible ordering compared to the typical routing of processes. Unlike regular processes, an ad-hoc subprocess does not contain a complete, structured BPMN2 diagram description, for example from start event to end event. Instead the ad-hoc subprocess contains only activities, sequence flows, gateways, and intermediate events. An ad-hoc subprocess can also contain data objects and data associations. The activities within the ad-hoc subprocesses are not required to have incoming and outgoing sequence flows. However, it is possible to specify sequence flows between some of the contained activities. When used, sequence flows provide the same ordering constraints as in a regular process. To have any meaning, intermediate events must have outgoing sequence flows and they can be triggered multiple times while the ad-hoc subprocess is active.
 
-An Ad-Hoc Sub-Process or Process contains a number of embedded inner Activities and is intended to be executed with a more flexible ordering compared to the typical routing of Processes. Unlike regular Processes, it does not contain a complete, structured BPMN diagram descriptioni.e., from Start Event to End Event. Instead the Ad-Hoc Sub-Process contains only Activities, Sequence Flows, Gateways, and Intermediate Events. An Ad-Hoc Sub-Process MAY also contain Data Objects and Data Associations. The Activities within the Ad-Hoc Sub- Process are not REQUIRED to have incoming and outgoing Sequence Flows. However, it is possible to specify Sequence Flows between some of the contained Activities. When used, Sequence Flows will provide the same ordering constraints as in a regular Process. To have any meaning, Intermediate Events will have outgoing Sequence Flows and they can be triggered multiple times while the Ad-Hoc Sub-Process is active.
 
+.Reusable subprocess
+Reusable subprocesses appear collapsed within the containing process, and its elements are not visible until you click a plus sign at the lower edge of the rounded rectangle.
```

Comment:
"its elements" Of the process or subprocess? I assume process since subprocesses is plural (would be "their elements"). Just checking.